input/templates/partials/catalog/mix-match-product3.json

```diff
@@ -0,0 +1,259 @@
+{
+    "list" : [
```

Comment:
Well at least I see you are using variants instead of products, right? But we have a `product-variant-1.json`, you can create more if you want.